cloud_testing/kotlin_poc/src/main/kotlin/ftl/run/AndroidTestRunner.kt

```diff
@@ -53,4 +46,33 @@ object AndroidTestRunner : GenericTestRunner {
 
         return afterRunTests(jobs, runGcsPath, stopwatch, config)
     }
+
+    /**
+     * If given a Gcs URI, then download the APKs
```

Comment:
Is this comment still valid? `then download the APKs` I see the upload logic however I'm not sure where the download code is in `resolveApks`